### ISSUE TITLE
Uncaught TypeError: Cannot read property 'suggestion_details' of null

### ISSUE THREAD

**Issue by mosaicer** [mosaicer/Muting_on_Twitter#10]
```javascript
if (
  targetNode.style.display !== 'none' &&
  JSON.parse(targetNode.getAttribute('data-suggestion-json')).suggestion_details.suggestion_type // => suggestion_details is null
) {
  targetNode.style.display = 'none';
  return;
}
```

**Comment by mosaicer** [comment]
Fixed on 1b54ad5.

**Comment by mosaicer** [comment]
When the JSON data made from the attribute `data-suggestion-json` data is null, this error occurs. Therefore, I make sure the JSON data is not null, on 09af4d1.
